File: C++ Code/Basics/Mensuration/3 D/Cube.cpp
// WAP to calculate Volume,TSA and LSA of a cube.
#include<iostream>
using namespace std;
int main(){
    float a;
    cout<<"Enter the length of edge of the cube : ";
    cin>>a;
    cout<<"Volume = "<<a*a*a<<endl;
    cout<<"Total Surface Area = "<<6*a*a<<endl;
    cout<<"Lateral Surface Area = "<<4*a*a;
    return 0;
}

File: Java Code/Basics/Mensuration/Cube.java
import java.util.*;

public class Cube{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.println("For the cube :");
    System.out.print("Enter edge length : ");
    float a=sc.nextFloat();
    float TSA=6*a*a;
    float LSA=4*a*a;
    float V=a*a*a;
    System.out.println("Lateral Surface Area = "+LSA);
    System.out.println("Total Surface Area : "+TSA);
    System.out.println("Volume : "+V);
  }
}